src/Grpc.Net.Client/GrpcChannelOptions.cs

```diff
@@ -0,0 +1,49 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using Grpc.Core;
+
+namespace Grpc.Net.Client
+{
+    /// <summary>
+    /// An options class for configuring a <see cref="GrpcChannel"/>.
+    /// </summary>
+    public sealed class GrpcChannelOptions
+    {
+        /// <summary>
+        /// Gets or sets the transport options for the channel. Transport options are required
+        /// to successfully create a channel.
+        /// </summary>
+        public GrpcTransportOptions? TransportOptions { get; set; }
```

Comment:
can the user currently use the TransportOptions to actually configure something? It looks like the transport options are only set when the channel builder is created and I don't see a situation where they'd need to be updated.
Perhaps this doesn't have to be a public property?

Reply:
They could do this: `TransportOptions = new HttpClientTransportOptions { HttpClient = httpClient }`